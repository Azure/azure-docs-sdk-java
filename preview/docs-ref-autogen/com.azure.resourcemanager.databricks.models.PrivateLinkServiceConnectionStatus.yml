### YamlMime:JavaType
uid: "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus"
fullName: "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus"
name: "PrivateLinkServiceConnectionStatus"
nameWithType: "PrivateLinkServiceConnectionStatus"
summary: "Defines values for PrivateLinkServiceConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum?displayProperty=fullName\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateLinkServiceConnectionStatus extends ExpandableStringEnum<PrivateLinkServiceConnectionStatus>"
constructors:
- "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus.PrivateLinkServiceConnectionStatus()"
fields:
- "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus.APPROVED"
- "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus.DISCONNECTED"
- "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus.PENDING"
- "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus.REJECTED"
methods:
- "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.databricks.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-databricks:1.0.0-beta.2
