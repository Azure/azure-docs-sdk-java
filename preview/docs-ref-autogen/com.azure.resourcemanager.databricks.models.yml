### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.databricks.models"
fullName: "com.azure.resourcemanager.databricks.models"
name: "com.azure.resourcemanager.databricks.models"
summary: "Package containing the data models for DatabricksClient. ARM Databricks."
classes:
- "com.azure.resourcemanager.databricks.models.AddressSpace"
- "com.azure.resourcemanager.databricks.models.CreatedBy"
- "com.azure.resourcemanager.databricks.models.CustomParameterType"
- "com.azure.resourcemanager.databricks.models.Encryption"
- "com.azure.resourcemanager.databricks.models.ErrorInfo"
- "com.azure.resourcemanager.databricks.models.ErrorInfoException"
- "com.azure.resourcemanager.databricks.models.KeySource"
- "com.azure.resourcemanager.databricks.models.ManagedIdentityConfiguration"
- "com.azure.resourcemanager.databricks.models.OperationDisplay"
- "com.azure.resourcemanager.databricks.models.OperationListResult"
- "com.azure.resourcemanager.databricks.models.PeeringProvisioningState"
- "com.azure.resourcemanager.databricks.models.PeeringState"
- "com.azure.resourcemanager.databricks.models.ProvisioningState"
- "com.azure.resourcemanager.databricks.models.Sku"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeeringList"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeeringPropertiesFormatDatabricksVirtualNetwork"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeeringPropertiesFormatRemoteVirtualNetwork"
- "com.azure.resourcemanager.databricks.models.WorkspaceCustomBooleanParameter"
- "com.azure.resourcemanager.databricks.models.WorkspaceCustomParameters"
- "com.azure.resourcemanager.databricks.models.WorkspaceCustomStringParameter"
- "com.azure.resourcemanager.databricks.models.WorkspaceEncryptionParameter"
- "com.azure.resourcemanager.databricks.models.WorkspaceListResult"
- "com.azure.resourcemanager.databricks.models.WorkspaceProviderAuthorization"
- "com.azure.resourcemanager.databricks.models.WorkspaceUpdate"
interfaces:
- "com.azure.resourcemanager.databricks.models.Operation"
- "com.azure.resourcemanager.databricks.models.Operations"
- "com.azure.resourcemanager.databricks.models.VNetPeerings"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.Definition"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages.Blank"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages.WithAllowForwardedTraffic"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages.WithAllowGatewayTransit"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages.WithAllowVirtualNetworkAccess"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages.WithDatabricksAddressSpace"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages.WithDatabricksVirtualNetwork"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages.WithRemoteAddressSpace"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages.WithRemoteVirtualNetwork"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.DefinitionStages.WithUseRemoteGateways"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.Update"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.UpdateStages"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.UpdateStages.WithAllowForwardedTraffic"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.UpdateStages.WithAllowGatewayTransit"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.UpdateStages.WithAllowVirtualNetworkAccess"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.UpdateStages.WithDatabricksAddressSpace"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.UpdateStages.WithDatabricksVirtualNetwork"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.UpdateStages.WithRemoteAddressSpace"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.UpdateStages.WithRemoteVirtualNetwork"
- "com.azure.resourcemanager.databricks.models.VirtualNetworkPeering.UpdateStages.WithUseRemoteGateways"
- "com.azure.resourcemanager.databricks.models.Workspace"
- "com.azure.resourcemanager.databricks.models.Workspace.Definition"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.Blank"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithAuthorizations"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithCreatedBy"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithManagedResourceGroupId"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithParameters"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithSku"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithStorageAccountIdentity"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithTags"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithUiDefinitionUri"
- "com.azure.resourcemanager.databricks.models.Workspace.DefinitionStages.WithUpdatedBy"
- "com.azure.resourcemanager.databricks.models.Workspace.Update"
- "com.azure.resourcemanager.databricks.models.Workspace.UpdateStages"
- "com.azure.resourcemanager.databricks.models.Workspace.UpdateStages.WithTags"
- "com.azure.resourcemanager.databricks.models.Workspaces"
metadata: {}
package: "com.azure.resourcemanager.databricks.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-databricks:1.0.0-beta.1
