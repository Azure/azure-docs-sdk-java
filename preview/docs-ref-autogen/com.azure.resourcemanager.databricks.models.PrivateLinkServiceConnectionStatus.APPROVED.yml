### YamlMime:JavaMember
uid: "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.databricks.models.PrivateLinkServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkServiceConnectionStatus."
  syntax: "public static final PrivateLinkServiceConnectionStatus APPROVED"
  desc: "Static value Approved for PrivateLinkServiceConnectionStatus."
type: "field"
metadata: {}
package: "com.azure.resourcemanager.databricks.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-databricks:1.0.0-beta.2
